script/utils/mento/GovernanceHelper.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.18;
+
+import { Script, console2 } from "forge-std/Script.sol";
+import { ICeloGovernance } from "../../interfaces/ICeloGovernance.sol";
+import { IGovernor } from "../../interfaces/IGovernor.sol";
+import { Chain } from "./Chain.sol";
+
+contract GovernanceHelper is Script {
+  struct MentoGovernanceTransaction {
+    address[] targets;
+    uint256[] values;
+    bytes[] calldatas;
+    string description;
+  }
+
+  function createProposal(
+    ICeloGovernance.Transaction[] memory transactions,
+    string memory descriptionURL,
+    address governance
+  ) internal {
+    if (Chain.isCelo()) {
+      verifyDescription(descriptionURL);
+    } else {
+      // Add timestamp to the description URL on testnets to avoid proposalId conflicts
+      descriptionURL = string(abi.encodePacked(descriptionURL, abi.encode(block.timestamp)));
+    }
+
+    MentoGovernanceTransaction memory govTx;
+
+    govTx.description = descriptionURL;
+    govTx.targets = new address[](transactions.length);
+    govTx.values = new uint256[](transactions.length);
+    govTx.calldatas = new bytes[](transactions.length);
+
+    for (uint256 i = 0; i < transactions.length; i++) {
+      govTx.targets[i] = transactions[i].destination;
+      govTx.values[i] = transactions[i].value;
+      govTx.calldatas[i] = transactions[i].data;
+    }
+
+    uint256 proposalId = IGovernor(governance).propose(govTx.targets, govTx.values, govTx.calldatas, govTx.description);
+
+    console2.log("Proposal was successfully created. ID: ", proposalId);
+  }
+
+  function simulateProposal(ICeloGovernance.Transaction[] memory transactions, address governance) internal {
```

Comment:
why are the transactions here of type ICeloGovernance.Transaction?

Reply:
the same tx structure is kept for the new governance so that we can keep writing the MUxx scripts the same way we used to. the main idea was to be able to use the repo for both governances with minimal changes and this structure allows us to use MUs and helper functionalities around that with both governances.